plans/feat-1786-load-average.md

```
@@ -0,0 +1,59 @@
+# #1786 — usage ゲージの横にマシンの load average を出す
+
+## 何を作るか
+
+グリッドヘッダの usage ゲージ（`claude 5h 42% 7d 13%`）の右に、サーバーを動かしている
+マシンの負荷を **コア数で正規化した %** で 1 トークン足す。
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)